create english docs and set to default (#3)
stack-spot/crystal-typescript-api-stack: README.md

# crystal-typescript-api-stack

> The **Typescript API** Stack makes it possible to create entirely serverless services, with Lambdas implemented in Typescript, automatic creation of API Gateway and other cloud features

Reviewer: ```suggestion
The **Typescript API** Stack creates an entirely serverless service, with Lambdas implemented in Typescript, automatic creation of API Gateway, and other cloud features.
```

> With the **Typescript API** you can develop and publish Lambdas based on the **OpenAPI** specification and fully integrated with your AWS environment. This allows you to automatically configure an **API Gateway**, which connects to all Lambdas generated from a given OpenAPI specification file.

Reviewer: ```suggestion
You can develop and publish Lambdas based on the **OpenAPI** specification and fully integrated with your AWS environment with **Typescript API**. This allows you to automatically configure an **API Gateway**, which connects to all Lambdas generated from a given OpenAPI specification file.
```

> The **Typescript API** Stack still uses an initial Template for a new project as the basis for creating the infrastructure in your AWS environment

Reviewer: ```suggestion
The **Typescript API** Stack still uses an initial Template for a new project, it works as a base to create the infrastructure in your AWS environment
```

> After the base Template, it is possible to use the **`OpenAPI Plugin`** plugin to add the ability to generate the base code of Lambdas, following the Hexagonal model. Thus, the code is ready for the development of business rules

Reviewer: ```suggestion
After the base Template, you can use the **`OpenAPI Plugin`** to add the ability to generate a base code of Lambdas, following the Hexagonal model. Thus, the code is ready for the business rules development. 
```

> Stack was developed following all good software architecture practices, such as

Reviewer: ```suggestion
The Stack was developed following the best software architecture practices, such as:
```
